libs/LedKit/include/internal/Sleeping.h

```diff
@@ -0,0 +1,43 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+// ? LCOV_EXCL_START - Exclude from coverage report
+
+#include "LEDAnimation.h"
+
+namespace leka::animation {
```

Comment:
animation c'est un peu générique, je mettrai 

```suggestion
namespace leka::led::animation {
```